includes/models/model.llms.order.php

```diff
@@ -647,14 +647,23 @@ public function get_customer_full_address() {
 	 * @since 3.0.0
 	 * @since 3.10.0 Unknown.
 	 * @since 5.3.1 Set the `post_date` property using `llms_current_time()`.
+	 * @since [version] Remove usage of deprecated `strftime()`.
 	 *
 	 * @param string $title Title to create the post with.
 	 * @return array
 	 */
 	protected function get_creation_args( $title = '' ) {
 
+		$ts = llms_current_time( 'timestamp' );
```

Comment:
I'm not totally sure of the implications of this change here.
This impacts on the `post_date`.
Can we keep passing `llms_current-time( 'mysql' )` as the `post_date` below?

Reply:
@eri-trabiccolo how bout this way?

Reply:
Seems good, thanks <3